test/tests/functional/pbs_ralter.py

```diff
@@ -391,11 +390,24 @@ def alter_a_reservation(self, r, start, end, shift,
                 msg = "Resv;" + r + ";Reservation alter confirmed"
             else:
                 msg = "Resv;" + r + ";Reservation alter denied"
-
-            lines = self.server.log_match(msg, n='ALL', allmatch=True,
-                                          interval=2, max_attempts=30)
-            self.assertEqual(len(lines), sequence)
-
+            attempts = 1
+            interval = 0.5
+            max_attempts = 30
+            while attempts <= max_attempts:
+                lines = self.server.log_match(msg, n='ALL', allmatch=True,
+                                              interval=1, max_attempts=30)
```

Comment:
This could take a long time.  Each log_match() can take up to 30 seconds, and you're calling log_match() up to 30 times.  That's 15m.  Also, if you don't match within the first 30 tries, log_match() will throw an exception.  This loop won't happen more than that.  Is this what you want?

Reply:
Thanks for catching this. 
Modified. Now the max try will take 50sec.